updates_and_signals/atomic_message_handlers/activities.py

```diff
@@ -0,0 +1,45 @@
+import asyncio
+from dataclasses import dataclass
+from typing import List
+
+from temporalio import activity
+
+
+@dataclass(kw_only=True)
```

Comment:
```suggestion
@dataclass
```

Probably not needed, but no big deal

Reply:
You can still use named arguments. We use them in lots of places, but since we're the only users of them we don't need to set this setting to force us to use them. Also, we have a CI check for our samples in 3.8 and I don't think this came about until 3.10 (we can look into relaxing our CI version constraints though).

Reply:
Ah, too bad.  I'd rather people who pattern-match off of this sample be directed toward best practices.  Will remove for now.  I wonder if we have stats on python versions people actually use in the wild?